pages/common/aapt.md

```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Android Asset Packaging Tool.
+> Compiles and packages an Android app’s resources.
+> More information: <https://developer.android.com/studio/command-line/aapt2>.
+
+- List files contained in an APK archive:
+
+`aapt list {{path/to/app.apk}}`
+
+- Display an app's metadata (version, permissions, etc.):
+
+`aapt dump badging {{path/to/app.apk}}`
+
+- Create a new APK archive with files from the specified directory:
+
+`aapt package -F {{path/to/app.apk}} {{path/to/directory}}`
```

Comment:
I couldn't find out what does `-F` stand for. Can we include that in the description ?

Reply:
It is how you specify the output APK file.

Reply:
I think it might be `File`, @agnivade? Not a standard convention, but I guess it works.

Reply:
Yeah, I assumed that. But then why is it beside the apk flag ? Shouldn't it be beside the directory token ?

Reply:
It is logic that `-F` comes before the APK filename, since it is used to specify the APK path.

Reply:
To put it differently, @agnivade, the `-F` option has nothing to do with the word "files" in the command description. It's just the name of the output `.apk` file to generate.

Reply:
Got it now. Thanks.